### ISSUE TITLE
Stratholme Scarlet Bastion trash scripting issues.

### ISSUE THREAD

**Issue by Azeron223** [slowtorta/turtlewow-bug-tracker#4948]
Inside Stratholme Scarlet Bastion, once you arrive in the Scarlet Crusade area, the non-elite skeletons in the area are just idling around all the Scarlet NPCs when these guys aren't usually considered friends.

The [Skeletal Guardian](https://database.turtle-wow.org/?npc=10390) (NPC ID: 10390) and [Skeletal Berserker](https://database.turtle-wow.org/?npc=10391) (NPC ID: 10391) should be constantly attacking the [Crimson Guardsman](https://database.turtle-wow.org/?npc=10418) (NPC ID: 10418) on the left and right sides of the Scarlet Bastion building just before you go inside.

There should be a maximum of 3 skeletons attacking each NPC at a time, though while each Scarlet Crusade NPC is still alive, the skeletons should slowly respawn from one of the 3 locations skeletons are currently idling in and engage this Scarlet Crusade mob in combat again, it should also generally be the case that the Scourge faction and the Scarlet Crusade faction are not friendly with each other and attack on sight.

**Comment by twowTurtReynolds** [comment]
Indeed, I remember from classic that they continually respawn and are easily defeated by the scarlets. Once players enter the area, the undead stop respawning.
